### PR TITLE
[BIT] 引擎补齐：在复测后的修复

### PR DESCRIPTION
**Pull request by Cutelemon6**
### 概览
修复或者检查了 merged_2 ~ merged_6，merged_11 中属于自己认领的报错，具体为
* merged_5 中 paddle.nn.functional.max_unpool1d, paddle.nn.functional.max_unpool2d, paddle.nn.functional.max_unpool3d，已修复
* paddle.Tensor.\_\_mul__，bf16 问题，已放入 fresh_report 
* paddle.Tensor.\_\_sub__，kernel 问题，已放入 fresh_report 
* paddle.clip，已修复
* paddle.incubate.nn.functional.fused_rms_norm，kernel unimplemented 问题，已放入 fresh_report 
* paddle.incubate.nn.functional.masked_multihead_attention，kernel 问题，已放入 fresh_report 
* paddle.nn.functional.feature_alpha_dropout，添加随机性 api 逻辑到 need_skip()
* paddle.nn.functional.poisson_nll_loss，bf16 问题，已放入 fresh_report 
* paddle.nn.functional.softmax，bf16 问题，已放入 fresh_report 
* paddle.normal，修复 skip 列表
* paddle.roll cuda kernel 问题，已放入 fresh_report 
* paddle.signal.istft 相关 Paddle 仓库 PR [#72282](https://github.com/PaddlePaddle/Paddle/pull/72282) 已于 4月18日合入，使用 0421 paddle库 develop 分支构建的安装包测试能够通过

### max_unpool{1d, 2d, 3d}
修复时调用了 max_pool{1, 2, 3}d_with_index 这个 kernel，这个 kernel 不支持 int64 的 x，所以遇到 int64 将其转换为 float64

### paddle.clip
PR #42 中对 clip 进行了不完全的 min 变量改名为 min_numpy_tensor，回测发现了这个问题

### paddle.nn.functional.feature_alpha_dropout
PR #40 中仅更新了随机性函数 api list，本 PR 添加随机性 api 逻辑到 need_skip()

### paddle.normal
skip 列表每个 str api 应该有 `,` 分隔，否则两个 str 会合并

### CODE REVIEW

File: tester/paddle_to_torch/mapping.json [hunk 1]
```diff
@@ -51,6 +51,16 @@
...
     },
+    "paddle.as_strided": {
+        "torch_api": "torch.as_strided",
+        "paddle_torch_args_map": {
+            "x": "input",
+            "shape": "size",
+            "stride": "stride",
+            "offset": "storage_offset"
+        },
+        "min_input_args": 3
```
Comment (cangtianhuang) [inline tester/paddle_to_torch/mapping.json:62]: 此参数可以不用加，没用上了

Comment (cangtianhuang) [inline tester/paddle_to_torch/mapping.json:62]:
```suggestion
        }
```

**Review by cangtianhuang — CHANGES_REQUESTED** [review]
此参数可以不用加，没用上了

**Review by yuwu46 — COMMENTED** [review]
api_config下的改动可以approve，paddle_to_torch部分请@cangtianhuang review

**Comment by cangtianhuang** [comment]
tester/api_config/api_config_accuracy_support_11.txt 可以 重命名为 tester/api_config/api_config_support2torch_2.txt
每个人认领两个文件：
api_config_support2torch_*.txt
api_config_paddleonly_*.txt

**Review by cangtianhuang — APPROVED** [review]
LGTM

**Review by yuwu46 — APPROVED** [review]

**Review by wanghuancoder — APPROVED** [review]
LGTM
